specification/cognitiveservices/OpenAI.Inference/models/chat.completions.tsp

```diff
@@ -176,75 +201,113 @@ model ChatCompletionsOptions {
     determines the intensity and risk level of harmful content) and if it has been filtered or not.
     """)
   @added(ServiceApiVersions.v2023_06_01_Preview)
   @projectedName("json", "content_filter_results")
   contentFilterResults?: ContentFilterResults;
 }
 
+@doc("The representation of a single prompt completion as part of an overall streaming chat completions request.")
+model ChatChoiceChunk {
+  ...ChatChoiceBase;
+
+  @doc("The delta message content for a streaming response.")
+  @projectedName("json", "delta")
+  delta: ChatMessageDelta;
+
+  #suppress "@azure-tools/typespec-azure-core/no-nullable" "The operation already returns nulls"
+  @doc("The reason that this chat completions choice completed its generated.")
+  @projectedName("json", "finish_reason")
+  finishReason: Nullable<CompletionsFinishReason>;
+}
+
 @doc("""
-Representation of the response data from a chat completions request.
-Completions support a wide variety of tasks and generate text that continues from or "completes"
-provided prompt data.
+The representation of a single prompt completion as part of an overall chat completions request.
+Generally, `n` choices are generated per provided prompt with a default value of 1.
+Token limits and other settings may limit the number of choices generated.
 """)
-model ChatCompletions {
+model ChatChoice {
+  ...ChatChoiceBase;
+
+  @doc("The chat message for a given chat completions prompt.")
+  @projectedName("json", "message")
+  message: ChatMessage;
+
+  @doc("The reason that this chat completions choice completed its generated.")
+  @projectedName("json", "finish_reason")
+  finishReason: CompletionsFinishReason;
+
+}
+
+@doc("Model representing the common base for chat completions response types.")
+model ChatCompletionsBase {
   @doc("A unique identifier associated with this chat completions response.")
   @projectedName("json", "id")
   id: string;
 
   @doc("""
-    The first timestamp associated with generation activity for this completions response,
-    represented as seconds since the beginning of the Unix epoch of 00:00 on 1 Jan 1970.
-    """)
+  The first timestamp associated with generation activity for this completions response,
+  represented as seconds since the beginning of the Unix epoch of 00:00 on 1 Jan 1970.
+  """)
   @projectedName("json", "created")
   @projectedName("java", "createdAt")
   @encode(DateTimeKnownEncoding.unixTimestamp, int32)
   created: utcDateTime;
 
+  @doc("""
+  Content filtering results for zero or more prompts in the request. In a streaming request,
+  results for different prompts may arrive at different times or in different orders.
+  """)
+  @added(ServiceApiVersions.v2023_06_01_Preview)
+  @projectedName("json", "prompt_annotations")
+  promptFilterResults?: PromptFilterResult[];
+}
+
+@doc("Representation of the response data from a streaming chat completions request.")
+model ChatCompletionsChunk {
```

Comment:
Should we consider still using the word "Streaming" somewhere, like `ChatCompletionsStreamingChunk`? Actual, not rhetorical question :)

Reply:
I'm not sure about that... we stream chunks but chunks are not the streaming. Also, I essentially lifted the name from the `object` property OAI returns.